tests/testthat/test-process_content.R

```diff
@@ -1,48 +1,43 @@
-test_that("processing content have proper output structure", {
-
-  my_project <-
-    initialize_project("gitai_test_project") |>
+test_that("[integration test] processing content have proper output structure", {
```

Comment:
Could you explain why we need to add this [integration test] prefixes to our tests?

Reply:
It's optional, just wanted to distinct the ones we're supposed to run with external keys.
I can revert it if you want.

Reply:
Got it. I was wondering if this is for some automatic processing I'm not aware about.
So I would remove those prefixes, as they seem to be nothing more then flag comments that can easily be out of sync with the code.  Instead (we don't have to do that in this PR though) I would add a helper function in the utils like:
skip_if_integration_tests_should_be_skipped() (or sth shorter but that is what I found most informative so far)
This function could be added at the beginning of all integration tests. Inside the function can have skip_on_cran() and additionally it can skip test if some environmental variable is set to TRUE, like GITAI_SKIP_INTEGRATION_TESTS.
It would be easy to switch off all integration tests with that while developing locally if needed or inside CI/CD.